src/player.cpp

```diff
@@ -12401,48 +12401,54 @@ void get_armor_on(player* p, body_part bp, std::vector<int>& armor_indices) {
 }
 
 void player::armor_absorb(damage_unit& du, item& armor) {
-    if (rng(0,100) <= armor.get_coverage()) {
-        if (armor.is_power_armor()) { // TODO: add some check for power armor
-        }
-
-        const float effective_resist = resistances(armor).get_effective_resist(du);
-        // Amount of damage mitigated
-        const float mitigation = std::min(effective_resist, du.amount);
-        du.amount -= mitigation; // mitigate the damage first
-
-        // Scale chance of article taking damage based on the number of parts it covers.
-        // This represents large articles being able to take more punishment
-        // before becoming inneffective or being destroyed.
-        const int num_parts_covered = armor.get_covered_body_parts().count();
-        if( !one_in( num_parts_covered ) ) {
-            return;
-        }
+    if (rng(0,100) > armor.get_coverage()) {
```

Comment:
Well crap, I fixed the issue where 100% coverage always had a 1% miss chance, but missed the fact that it had the same error on the other end.  I'm not sure there is anything with 0% coverage though, "trinkets" with no protection and no encumbrance are generally not assigned to a body part at all.  It'd be pretty funny if there was some "lucky charm" that really did have a 1% chance of intercepting some damage though.